src/main/java/com/minecolonies/coremod/entity/pathfinding/pathjobs/PathJobRandomPos.java

```diff
@@ -163,6 +168,55 @@ protected double getNodeResultScore(@NotNull final MNode n)
         return 0;
     }
 
+    @Override
+    protected double computeCost(
+      @NotNull final BlockPos dPos,
+      final boolean isSwimming,
+      final boolean onPath,
+      final boolean onRails,
+      final boolean railsExit,
+      final boolean swimStart,
+      final boolean corner,
+      final BlockState state,
+      final BlockPos blockPos)
+    {
+        double cost = Math.sqrt(dPos.getX() * dPos.getX() + dPos.getY() * dPos.getY() + dPos.getZ() * dPos.getZ());
```

Comment:
what is this for?

Reply:
Removed door cost from random pathing, this made them avoid doors, soooo much, they would never enter the bedrooms =D